components/cam/src/physics/crm/sam/ADV_MPDATA/advect_scalar2D.F90

```diff
@@ -105,8 +130,11 @@ subroutine advect_scalar2D (ncrms, f, u, w, rho, rhow, flux)
         enddo
       enddo
     endif  ! nonos
-
+#if defined(_OPENACC)
     !$acc parallel loop collapse(3) async(asyncid)
```

Comment:
OpenACC directives are by default ignored if the compiler isn't OpenACC-capable and OpenACC isn't enabled. So there's no need to wrap these in `#ifdef`s I don't think.

Reply:
Totally agree, we shouldn't be controlling this with ifdefs.